package.json

```diff
@@ -68,21 +68,11 @@
         "command": "tacosontitan.toolbox.azure.devops.createDefaultTasks",
         "title": "Hazel's Toolbox: Create Default Azure DevOps Tasks"
       },
-      {
-        "command": "tacosontitan.toolbox.azure.devops.setWorkItem",
-        "title": "Hazel's Toolbox: Set Work Item",
```

Comment:
Let's keep this command and rename the title and command name to search instead of set. We're going to need the ability to search for work items at some point.